internals/secrethub/run.go

```diff
@@ -133,8 +135,25 @@ func (cmd *RunCommand) Run() error {
 		}
 	}
 
+	var parser tpl.Parser
+	switch cmd.templateVersion {
+	case "auto":
+		// Leave it up to the env file parser to detect the template version to use;
+		// The latest version is used for .env formatted environment files, v1 is used
+		// for yml formatted environment files.
+		parser = nil
+	case "1":
```

Comment:
In the command reference we say `v1` is that what we want or `1`? I have a light preference for `v1`. Or do we want both?

Reply:
Let's accept both :+1: